src/flexbox.rs

```diff
@@ -225,8 +228,8 @@ impl Forest {
         };
 
         let node_inner_size = Size {
-            width: node_size.width - padding_border.horizontal_axis_sum(),
-            height: node_size.height - padding_border.vertical_axis_sum(),
+            width: Some(node_size.width.unwrap_or_default() - padding_border.horizontal_axis_sum()),
+            height: Some(node_size.height.unwrap_or_default() - padding_border.vertical_axis_sum()),
```

Comment:
```suggestion
            width: node_size.width.map(|v| v - padding_border.horizontal_axis_sum()),
            height: node_size.height..map(|v| v -  - padding_border.vertical_axis_sum()),
```

Reply:
I think most of the issue are due to this problem that you repeated a few places:
you replaced undefined - value which is still undefined
by none.unwrap_or(0) - value which is -value

Reply:
Ah, I see :)